pyquil/tests/test_parser.py

```diff
@@ -395,13 +395,22 @@ def test_def_circuit():
     H a
     CNOT a b
 """.strip()
+
     defcircuit_no_qubits = """
 DEFCIRCUIT bell:
     H 0
     CNOT 0 1
 """.strip()
+
+    defcircuit_param = """
```

Comment:
@notmgsk Decided to leave the test case decoupled after looking at it more closely